common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerValidator.java

```diff
@@ -222,4 +222,14 @@ public static String getBrokerRedirectUri(final Context context, final String pa
         return PackageHelper.getBrokerRedirectUrl(packageName,
                 signatureDigest);
     }
+
+    /**
+     * Helper method to get MSAL Redirect Uri
+     *
+     * @param clientId
+     * @return String : MSAL Redirect Uri
+     */
+    public static String getMsalRedirectUri(@NonNull final String clientId){
```

Comment:
I was under the impression that MSAL with V2 broker's redirect URI should always be of the form msauth://<package_name>/<sign_hash>. Did the requirement change ?

Reply:
I'm not certain about this, but this redirect URL pattern is available as an option in the portal. We're also using it in the test app (which is why I was hitting this error).

Might be worth bringing this up during the standup.

Reply:
The scheme suggested in the _new portal_ should now match the broker redirect uri

```
msauth://<package_name>/<signature_hash_decoded>
```

Reply:
@danieldobalian can link to new docs

Reply:
Does this mean we're not supporting "msal" + clientId + "://auth"; anymore?

If yes, then I'll drop this PR.

Reply:
Yes that redirect is only supported for just standalone MSAL, not with broker.